tests/integrations/realcluster/deploy.sh

```diff
@@ -19,15 +21,17 @@ if [ ! -d "bin" ] || [ ! -e "bin/tikv-server" ] && [ ! -e "bin/tidb-server" ] &&
 	color-green "downloading binaries..."
 	color-green "this may take a few minutes, you can also download them manually and put them in the bin directory."
 	make pd-server WITH_RACE=1
-	$TIUP_BIN_DIR playground nightly --kv 3 --tiflash 1 --db 1 --pd 3 --without-monitor --tag pd_test \
-		--pd.binpath ./bin/pd-server \
+	$TIUP_BIN_DIR playground nightly --kv 3 --tiflash 1 --db 1 --pd 3 --without-monitor --tag pd_real_cluster_test \
+		--pd.binpath ./bin/pd-server --pd.config ./tests/integrations/realcluster/pd.toml \
 		> $CUR_PATH/playground.log 2>&1 &
 else
   # CI will download the binaries in the prepare phase.
   # ref https://github.com/PingCAP-QE/ci/blob/387e9e533b365174962ccb1959442a7070f9cd66/pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy#L55-L68
 	color-green "using existing binaries..."
+	make pd-server WITH_RACE=1
```

Comment:
Please remove this line, as the comment above explains that there is no need to `make pd` :)